quartz/plugins/transformers/ofm.ts

```diff
@@ -213,6 +217,35 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>>
         })
       }
 
+      if (opts.inlineFootnotes) {
+        // Replaces ^[inline] footnotes with regular footnotes [^1]:
+        const footnotes: Record<string, string> = {}
+
+        // Replace inline footnotes with references and collect definitions
+        const result = (src as string).replace(
+          inlineFootnoteRegex,
+          (_match: string, content: string) => {
+            const id = `inline-${Object.keys(footnotes).length + 1}`
+            footnotes[id] = content.trim()
+            return `[^${id}]`
+          },
+        )
+
+        // Append footnote definitions if any are found
+        if (Object.keys(footnotes).length > 0) {
+          return (
+            result +
```

Comment:
My concern with this is that we are dumping this at the bottom of the file. Which in some cases it might mess up with custom parsers

probably just need to warn people about this.